index.html

```diff
@@ -20,160 +20,165 @@
   </head>
 
   <body>
-    <div class="header">
+    <section class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <section>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </section>
+
+      <button class="dovo">Donate Now</button>
+    </section>
+
+    <section class="content">
+      <section class="main">
+        <section class="alert">
+          You've helped us give <span class="text-highlight">72</span> bikes to
+          refugees and asylum seekers so far. <strong>Thank you!</strong>
+        </section>
+
+        <section class="hero">
+          <h1>Bikes for Refugees</h1>
+          <p>
+            Providing donated bikes and accessories to refugees and asylum
+            seekers in Scotland.
+          </p>
+          <button class="dovo">Donate a bike today</button>
+          <button class="vol">Volunteer</button>
+        </section>
+
+        <section>
+          <h2 class="heading-underline">Learn more</h2>
+
+          <section id="art">
```

Comment:
This would be a great place to use the [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) element